Cod3rsGrowth.Web/wwwroot/controller/CriarEditarEmpresas.controller.js

```diff
@@ -60,24 +77,67 @@ sap.ui.define([
 			});
 		},
 		_retornaValoresEmpresa() {
-			
 			const modelo = this.getView().getModel();
```

Comment:
pra que todo esse objeto que vc esta criado se vc fizer this.getView().getModel().getData() ja retorna o objeto